db/migrate/20160309182642_remove_social_events_table.rb

```diff
@@ -0,0 +1,134 @@
+class RemoveSocialEventsTable < ActiveRecord::Migration
```

Comment:
I would very much like us to stop doing these migrations and switch to [rake based data migrations](https://robots.thoughtbot.com/data-migrations-in-rails).